kfdef/kfctl_k8s_istio.v1.1.0.yaml

```diff
@@ -86,11 +86,6 @@ spec:
         name: manifests
         path: stacks/kubernetes/application/spartakus
     name: spartakus
-  - kustomizeConfig:
```

Comment:
If we remove tensorboard from manifests, can you also remove folders `stacks/kubernetes/application/tensorboard`?

Reply:
`stacks/kubernetes/application/tensorboard` never seemed to have existed, so we should be good there. :)